src/clients/dns.py

```diff
@@ -79,3 +79,30 @@ def is_keycloak_dr_up_and_receiving_traffic(ip: str):
         return 'gold_up'
     else:
         return 'error'
+
+
+def fetch_domain_by_env(env, prod):
+    switcher = {
+        "sandbox-dev": "dev.sandbox.loginproxy.gov.bc.ca",
+        "sandbox-test": "test.sandbox.loginproxy.gov.bc.ca",
+        "sandbox-prod": "sandbox.loginproxy.gov.bc.ca",
+        "dev": "dev.loginproxy.gov.bc.ca",
+        "test": "test.loginproxy.gov.bc.ca",
+        "prod": "loginproxy.gov.bc.ca",
+    }
+    if prod:
+        return switcher.get(env, "")
+    else:
+        return switcher.get("%s-%s" % ("sandbox", env), "")
+
+
+def check_dns_by_env(env, mode):
+    print('hello')
```

Comment:
print hello

Reply:
hahaha that got through some how. Removed it